api/api/settings.py

```diff
@@ -43,18 +43,22 @@
     'oauth2_provider',
     'social_django',
     'drf_social_oauth2',
+    'corsheaders',
 ]
 
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+CORS_ALLOWED_ORIGINS = os.environ.get('ALLOWED_HOSTS', 'http://127.0.0.1:8000').split(',')
```

Comment:
Default should be empty string. AFAIK, we don't need the port in origins, do we?

Reply:
Done.